### ISSUE TITLE
query about starting the structural variation pipeline

### ISSUE THREAD

**Issue by prasundutta87** [bcbio/bcbio-nextgen#3623]
Hi,

I am planning to use the germline structural variation pipeline from bcbio-nextgen for calling germline SVs. I was unable to follow the what was mentioned here and was unable to get a starting point -https://bcbio-nextgen.readthedocs.io/en/latest/contents/structural_variants.html

I have already aligned CRAM files and was wondering if I can be directed towards how can I start using the pipeline (like get yaml files and so on).

Regards,
Prasun

**Comment by naumenko-sa** [comment]
Hi @prasundutta87 !

Could you please outline what have you tried and what did not work for you?

Did you install bcbio first of all?
Did you run the structural variant calling example?

bcbio supports cram as input as well, even though there are some limitations:
https://github.com/bcbio/bcbio-nextgen/issues/3011

Sergey

**Comment by prasundutta87** [comment]
Hi Sergey, 

Thanks for your reply. I had actually installed bcbio in 2020, but I have recently started using it now starting with SV specific pipeline. I have tried working on the example dataset, however, it gets stuck in the CollectSequencingArtifactMetrics stage in which I get an error named "ERROR: Invalid argument '-REFERENCE_SEQUENCE'". How do I resolve this issue? This is the exact command which is being executed:

`unset JAVA_HOME && export PATH=/gpfs/igmmfs01/eddie/aitman-lab/Prasun/bcbio_nextgen/bcbio_data/anaconda/bin:"$PATH" && picard -Xms750m -Xmx3500m -XX:+UseSerialGC CollectSequencingArtifactMetrics -REFERENCE_SEQUENCE /gpfs/igmmfs01/eddie/aitman-lab/Prasun/bcbio_nextgen/bcbio_data/genomes/Hsapiens/hg38/seq/hg38.fa -INPUT /gpfs/igmmfs01/eddie/aitman-lab/Prasun/NA12878-sv-eval/work/align/NA12878/NA12878-sort.bam -OUTPUT /gpfs/igmmfs01/eddie/aitman-lab/Prasun/NA12878-sv-eval/work/bcbiotx/tmpnvoleqan/NA12878/NA12878 --VALIDATION_STRINGENCY SILENT`

The issues what I generally face with bcbio pipelines, is that, as the pipeline runs many internal scripts, without closely tracking the lines in the bcbio-nextgen-commands.log file, it become difficult to understand what all steps are exactly happenning. The yaml file gives an overall picture, but still many internal step descriptions are still missing. Is the command log file still the only way to understand what's actually happening under the hood?

I was also wondering if the bcbio piepline installs all the required tools automatically for a pipeline when the pipeline runs? For example, from the provenance folder in the work directory?

I have actually managed to convert my CRAMs to paired end fastq.gz and I can use them in the pipeline.

Regards,
Prasun
